test/unit/video.test.js

```diff
@@ -117,19 +116,13 @@ QUnit.test('should add the value to the languages object with lower case lang co
                   'should also match');
 });
 
-QUnit.test('should expose plugin registry function', function(assert) {
-  const pluginName = 'foo';
-  const pluginFunction = function(options) {};
-
-  assert.ok(videojs.plugin, 'should exist');
-
-  videojs.plugin(pluginName, pluginFunction);
-
-  const player = TestHelpers.makePlayer();
-
-  assert.ok(player.foo, 'should exist');
-  assert.equal(player.foo, pluginFunction, 'should be equal');
-  player.dispose();
+QUnit.test('should expose plugin functions', function(assert) {
+  assert.strictEqual(typeof videojs.registerPlugin, 'function');
```

Comment:
I dunno, I feel like these are pretty self-explanatory. Is there some value I'm not understanding in providing descriptions for things like this?